### DISCUSSION TITLE
Joins call and doesn't play audio

### DISCUSSION THREAD

**Discussion by Charbrose** [codetheweb/muse#1103] [Q&A]
Title is self explanatory about the issue. I transferred a working set of the files from my main pc (where it works) to my server pc. It was having issues with ffmpeg but I managed to get around that with a fresh install of ffmpeg. Now it joins call but doesn't play audio. I then decided to redo the files entirely and it provided the same issue. I'm open to any suggestions for a possible fix. Thank you in advance.

**Comment by Charbrose** [comment]
Follow up as for what the issue was. Wrong version of ffmpeg and node were installed on my server.

(versions that fixed it for me)
ffmpeg: ffmpeg version N-113781-g76b2bb96b4-20240222
node js: node v20.17.0

**Reply by MikeTodd** [reply]
I'm getting "ope: FFmpeg/avconv not found!" -- is that the same error you got? I'm on Node v22.14.0 and ffmpeg N-121746-gf39884b3fd-20251112 -- not sure how to get an older version of that if I need it.
